example.env

```diff
@@ -0,0 +1,5 @@
+MM_TEAM="test-team"
```

Comment:
(question) Since the `add_users.sh` script creates a team called botsample, would it make sense to have that team as a default in the example.env file?

Reply:
Sure